doc/guides/test/sanitycheck.rst

```diff
@@ -341,7 +341,7 @@ extra_configs: <list of extra configurations>
             depends_on: adc
           test_resolution_6:
             extra_configs:
-              - CONFIG_ADC_QMSI_SAMPLE_WIDTH=6
+              - CONFIG_LOG_DEFAULT_LEVEL=3
```

Comment:
how did an ADC related Kconfig turn into a LOG_LEVEL?

Reply:
I just picked that random symbol that exists. What's a better example I could use?

Reply:
@nashif *ping*

Note that this is in documentation.

Reply:
@nashif 

Changed it to this. Is that fine?

```
common:
  tags: drivers adc
tests:
  test:
    depends_on: adc
  test_async:
    extra_configs:
      - CONFIG_ADC_ASYNC=y
```